spec/factories/profile.rb

```diff
@@ -20,5 +20,38 @@
         association(:profile_os_minor_version, os_minor_version: os_minor_version)
       end
     end
+
+    trait :with_values do
+      transient do
+        value_count { 5 }
```

Comment:
The `value_count` variable renders the `with_values` obsolete ... please eliminate the `trait` similarly to rulegroups.